src/components/DropdownStyles.css

```diff
@@ -0,0 +1,148 @@
+/* Dropdown Button */
+.dropbtn {
+flex-direction: row;
+justify-content: center;
+align-items: center;
+padding: 0px;
+gap: 8px;
+width: 98px;
+height: 40px;
+background: #006D7D;
+border-radius: 8px;
+flex: none;
+order: 0;
+align-self: flex-start;
+flex-grow: 0;
+border: none;
+}
+
+/* parent div: move this if you want to relocate the button */
+.dropdown {
+   position: absolute;
```

Comment:
Remove this line

Reply:
Sorry which line? Should I remove the spaces between the end brackets and new CSS elements?

Reply:
line 21: position: absolute
We want to be able to position the button later on

Reply:
got it, you mentioned that before, must have missed that line